apps/builder/app/builder/features/sidebar-left/panels/pages/page-settings.tsx

```diff
@@ -1505,11 +1504,22 @@ const updatePage = (pageId: Page["id"], values: Partial<Values>) => {
       return;
     }
 
+    if (pages.homePage.id === pageId) {
+      updatePageMutable(pages.homePage, values, pages.folders);
+    }
+
+    for (const page of pages.pages) {
```

Comment:
seems like you would rather want to use .find or at least a break; once the page is found, .find is kinda making more semantical sense

Reply:
I just moved that code lines, there are no new code, just order fix

Reply:
iknow)

Reply:
honestly it's not better, for "semantics" we need .get(id)

Reply:
.find() returns one element, its semantically clearly better, perf doesn't really matter here, since you won't have thouthands pages